cdap-unit-test/pom.xml

```diff
@@ -123,6 +123,27 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-data-fabric</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
```

Comment:
for ``KafkaTester``? If so, should we move that class to a better place? Not for this PR though.